dev/run-tests.py

```diff
@@ -55,6 +55,10 @@ def determine_modules_for_files(filenames):
     for filename in filenames:
         if filename in ("appveyor.yml",):
             continue
+        if (os.environ.get("AMPLAB_JENKINS") and
+                filename in (".github/workflows/build_and_test.yml",
```

Comment:
I think even we can just ignore running build for entire `.github` directory.

Reply:
sounds good.